Flagsmith.FlagsmithClient/Interfaces/IFlag.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace Flagsmith.Interfaces
+{
+    public interface IFlag
+    {
+        int Id { get; }
+        bool Enabled { get; }
+        string Value { get; }
+        IFeature Feature { get; }
+
+        [Obsolete("Use 'Feature.Id' property, method will be removed in next version.")]
```

Comment:
Since we're creating a new major version here anyway, we can remove these methods and 'Obsolete' statements, right?